cmd/root.go

```diff
@@ -81,11 +81,6 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/"+pxDefaultDir+"/"+pxDefaultConfigName+")")
 	rootCmd.PersistentFlags().StringVar(&cfgContext, "context", "", "Force context name for the command")
 
-	// TODO: move these flags out of persistent
-	rootCmd.PersistentFlags().StringP("output", "o", "", "Output in yaml|json|wide")
```

Comment:
Please do not remove all flags without re adding them in appropriate commands.

Reply:
Yup just after the commit I realized it, pulled it back.